### ISSUE TITLE
Textarea blinks before TinyMCE rendering

### ISSUE THREAD

**Issue by dimkaufo** [HurricaneJames/react-tinymce-input#3]
I've found that you have `setTimeout` here, and component blinks with `textarea` before it shows an editor, could you make this setTimeout optional?

```
componentDidMount: function componentDidMount() {
    this.initTimeout = setTimeout(this.initTinyMCE, 100);
    this.initStartTime = Date.now();
}
```

**Comment by HurricaneJames** [comment]
Calling `initTinyMCE` immediately, even if `tinymce` is available does not fix the flicker, but I added a check to see if it is possible to bypass the setTimeout anyway.

What did work was pseudo-hiding the textarea element so it is not displayed on screen before tinymce creates its window.

I just released v1.0.5 that has both changes. Let me know if that works better for you.

**Comment by dimkaufo** [comment]
Thanks for the fast fix @HurricaneJames, now it works better
